packages/react/src/components/Select/Select.js

```diff
@@ -222,7 +212,13 @@ Select.propTypes = {
    * `true` to use the light version. For use on $ui-01 backgrounds only.
    * Don't use this to make tile background color same as container background color.
    */
-  light: PropTypes.bool,
+  light: FeatureFlags.enabled('enable-v11-release')
+    ? deprecate(
+        PropTypes.bool,
+        'The `light` prop for `Select` is no longer needed and has ' +
+          'been deprecated in v11. It will be moved in the next major release.'
```

Comment:
I think the `light` prop warnings should include a link or instruction to use `Layer`. Maybe we link to somewhere in the migration guide?

Reply:
I'll update! 👍🏽